ui-components/src/page_components/listing/ListingCard.tsx

```diff
@@ -17,6 +17,7 @@ export interface CardHeader {
   href?: string
   customClass?: string
   styleType?: AppearanceStyleType
+  pillType?: boolean
```

Comment:
since this is a boolean we might want to call it something like `isPillType` to help denote what the value should be. However, it's better to keep us consistent in the naming convention so whatever name the team would prefer I'm ok with.

Reply:
I liked that! Updated 👍